distribution/ddf-common/src/main/resources/etc/templates/example.ManagedService.config

```diff
@@ -1,22 +1,42 @@
-property.string="string"
-property.boolean.true=B"true"
-property.boolean.false=B"false"
-property.int=I"10"
-property.long=L"100"
-// Floats and doubles are supported using the IEEE 754 format. See the Documentation for more details.
-property.float=F"0x41280000"
-property.double=D"0x42c83f2e"
+property.string = "string"
+property.boolean.true = B"true"
+property.boolean.false = B"false"
+property.int = I"10"
+property.long = L"100"
+# Floats and doubles are supported using the IEEE 754 format. See the Documentation for more details.
+# F"10.5"
+property.float = F"1065353216"
+# D"100.1234"
+property.double = D"4636745974857667812"
 
-property.array.strings=["A","B","C"]
-property.array.booleans=B["true","false"]
-property.array.ints=I["10","20","30"]
-property.array.longs=L["100","200","300"]
-property.array.floats=F["0x3f8ccccd","0x400ccccd","0x40533333"]
-property.array.doubles=D["0x3f8fbe77","0x400ef9db","0x4056147b"]
-
-property.vector.strings=("A","B","C")
-property.vector.booleans=B("true","false")
-property.vector.ints=I("10","20","30")
-property.vector.longs=L("100","200","300")
-property.vector.floats=F("0x3f8ccccd","0x400ccccd","0x40533333")
-property.vector.doubles=D("0x3f8fbe77","0x400ef9db","0x4056147b")
\ No newline at end of file
+property.array.strings = [ \
```

Comment:
❓ Do we _have_ to put the values on different lines and use `\` at the end? If so, that's something we should add to the documentation.

Reply:
No, it is not required to put the values on different lines. However, if the values are split over multiple lines, `\` **must** be used. I added a note to the documentation:
>When properties are split over multiple lines for readability, end of lines must be specified with a backslash (`\`). See the examples for lists in the table.